script/utils/sim-sequence.sh

```diff
@@ -0,0 +1,308 @@
+#!/bin/bash
+
+
+LOGFILE="/tmp/sim-sequence.log"
+## Nonce Values
+MAX_NONCE_ERROR=9999999
+FUS_BEFORE=$MAX_NONCE_ERROR
+FOS_BEFORE=$MAX_NONCE_ERROR
+SC_BEFORE=$MAX_NONCE_ERROR
+L1PAO_BEFORE=$MAX_NONCE_ERROR
```

Comment:
What do we initialize all nonces to 9999999?

Reply:
will comment but because a safe can have the nonce=0 for example in sepolia or with a new safe. 
if the nonce is 0, we can't really know if this an error because the RPC is dead or retrived incorrect data.
but the if the nonce `9999999` is returned we no for sure that no value has been overwritten.